inst/WORDLIST

```diff
@@ -44,3 +44,15 @@ reproducibility
 reproducibly
 sd
 ungeneralized
+adae
+ADAE
+qc
+xpt
+Haesendonckx
+Heng
+Leng
+Ning
+Peikun
+Stackhouse
+Yilong
+Zhang  
```

Comment:
Just for my own knowledge, is this a wordlist so that the spellcheck will ignore?